cmd/cli/serve/util.go

```diff
@@ -145,12 +144,18 @@ func getIPFSConfig() (types.IpfsConfig, error) {
 	if err := config.ForKey(types.NodeIPFS, &ipfsConfig); err != nil {
 		return types.IpfsConfig{}, err
 	}
+
 	if ipfsConfig.Connect != "" && ipfsConfig.PrivateInternal {
+		ipfsConfig.PrivateInternal = false
+		log.Debug().Msg("disabling ipfs private internal")
+	}
+
+	/*if ipfsConfig.Connect != "" && ipfsConfig.PrivateInternal {
 		return types.IpfsConfig{}, fmt.Errorf("%s cannot be used with %s",
 			configflags.FlagNameForKey(types.NodeIPFSPrivateInternal, configflags.IPFSFlags...),
 			configflags.FlagNameForKey(types.NodeIPFSConnect, configflags.IPFSFlags...),
 		)
-	}
+	}*/
```

Comment:
Should probably just delete this commented out block. 

This is okay as a temporary measure, but once https://github.com/bacalhau-project/bacalhau/pull/3712 is completed there will be no PrivateInternal anyway

Reply:
Done

Reply:
Done